### ISSUE TITLE
Feature request to analysis health data

### ISSUE THREAD

**Issue by Kibria2017** [jasp-stats/jasp-desktop#1814]
1. (a) non-linear, (b) ordinal, (c) conditional logistic regression and (d) multinomial logistic Regression
2. Poisson Regression including (a) negative binomial, (b) zero-inflated and (c) zero truncated Poisson regression
3. Survival Analyses - (a) log-rank test, (b) kaplan-meyer and (c) cox proportional hazard

Thanks

**Comment by EJWagenmakers** [comment]
All of these are excellent suggestions.
E.J.

**Comment by Farscape1** [comment]
Thanks for adding the logsitic regression to  v .8.3. Any plans to add multinomial logistic regression. 

Thanks,

**Comment by EJWagenmakers** [comment]
Absolutely. @vankesteren and I should discuss the options. Are there any R packages you like in particular?

**Comment by Farscape1** [comment]
I like the Ripley's nnet. 

Thanks

**Comment by drhallak** [comment]
I'd also love to have the survival analysis / cos regression functionality for modeling organizational attrition. 

(a) log-rank test, (b) kaplan-meyer and (c) cox proportional hazard
